crates/allocator/src/lib.rs

```diff
@@ -191,5 +192,7 @@ mod test {
 
         let end = reader.current();
         assert_eq!(0, end.total_allocated() - start.total_allocated());
+
+        println!("{some_sum} cause some_vec to be used and not elided by dead-code analysis");
```

Comment:
Yep, it's needed to coerce the compiler into believing that the result is actually used, and it can't elide the whole thing away.

I'd prefer to keep this as `println!` since that's part of the standard library, this is inside a test case, and there isn't a current dependency on `tracing` from this crate.